Python/specialize.c

```diff
@@ -895,57 +902,65 @@ static int specialize_attr_loadclassattr(PyObject* owner, _Py_CODEUNIT* instr, P
     PyObject* descr, DescriptorClassification kind, bool is_method);
 static int specialize_class_load_attr(PyObject* owner, _Py_CODEUNIT* instr, PyObject* name);
 
-void
-_Py_Specialize_LoadAttr(_PyStackRef owner_st, _Py_CODEUNIT *instr, PyObject *name)
+/* Returns true if instance may have attribute `name`. */
```

Comment:
may in the sense of maybe, or in the sense of can?

Reply:
Neither of those really. I'll clarify the comment.